### PR TITLE
Hdrp/add border in shadow cascade

### PR DESCRIPTION
**Pull request by RSlysz**
### Purpose of this PR
Update the toggle for advanced mode in inspectors.
Add cascade visualizer, supporting disabled handles when not overriding.
image

image

---
### Release Notes
Update the toggle for advanced mode in inspectors.
Add cascade visualizer, supporting disabled handles when not overriding.

---
### Testing status
**Katana Tests**: https://katana.bf.unity3d.com/projects/com.unity.render-pipelines/builders?ScriptableRenderLoop_branch=HDRP%2Fadd-border-in-shadow-cascade&automation-tools_branch=add-platform-filter&unity_branch=trunk#

**Manual Tests**: Tested locally

**Automated Tests**: 

---
### Overall Product Risks
**Technical Risk**: Low-Medium

**Halo Effect**: Low

---

### CODE REVIEW

**Comment by sebastienlagarde** [comment]
THink to add katana link

File: com.unity.render-pipelines.high-definition/Editor/Lighting/Shadow/ShadowCascadeGUI.cs [hunk 1]
```diff
@@ -0,0 +1,404 @@
...
+                        //if (s_RestoreSceneView == null)
+                        //{
+                        //    s_RestoreSceneView = SceneView.lastActiveSceneView;
+                        //    if (s_RestoreSceneView != null)
+                        //    {
+                        //        s_OldSceneDrawMode = s_RestoreSceneView.cameraMode;
+                        //        s_OldSceneLightingMode = s_RestoreSceneView.sceneLighting;
+                        //        s_RestoreSceneView.cameraMode = SceneView.GetBuiltinCameraMode(DrawCameraMode.ShadowCascades);
+                        //    }
+                        //}
```
Comment (Chman) [inline com.unity.render-pipelines.high-definition/Editor/Lighting/Shadow/ShadowCascadeGUI.cs]: Any reason why we keep these commented-out sections of code?

**Comment by RSlysz** [comment]
@alelievr PR just a check up: does border are really percent? I interpreted them as such border in [0, 1] but there is no clamping. 
Also, when you'll fix cascade, reactivate the border too. I've not changed the static as it is not working at the moment.

File: com.unity.render-pipelines.high-definition/Editor/Lighting/Shadow/ShadowCascadeGUI.cs (hunk 1 above)
Comment (RSlysz) [reply com.unity.render-pipelines.high-definition/Editor/Lighting/Shadow/ShadowCascadeGUI.cs]: @sebastienlagarde On Unity render pipeline, they enter in cascade visualisation mode when editing the cascades' partitioning. (The above commented code). Do we need to add this too on your side?
By the way, there seams to have an issue with the shader displaying cascade as everything is white. (I suppose we must also match the color in the partition (unity render pipeline colors) to the colors in the scene when visualizing the cascade?)

**Comment by alelievr** [comment]
Yes, the border is in percent between 0 and 1. I'll enable the border when i fix them, thanks :)
Also i just double checked the blend direction and it's inward: it means that the border 0 will blend the cascade 1 into the 0, so it's good 👍.

**Pull request merged**